apps/ideaspace/src/components/modules/WorkshoppingPage/WorkshoppingPage.js

```diff
@@ -49,4 +66,4 @@ function WorkshoppingPage() {
   );
 }
 
-export default WorkshoppingPage;
+// export default WorkshoppingPage;
```

Comment:
we should be able to delete this line altogether with the change you made on line 21